<launch>
    <arg name="node_start_delay1" default="2.0" />
    <node name="offboard_mpc_controller" type="offboard_sim_mpc_controller.py" pkg="itm_nonlinear_mpc" output="screen">
        <rosparam file="$(find itm_nonlinear_mpc)/resources/itm_nonlinear_mpc_sim_sitl.yaml" />
        <!-- <param name="lib_path" value="$(find itm_nonlinear_mpc)/scripts/non_ros_casadi/nmpc.so" /> -->
        <remap from="/local_position_pose" to="/robot_pose" />
        <!-- <param name="mpc_c_file" value="$(find itm_nonlinear_mpc)/scripts/non_ros_casadi/mpc_v1.c" /> -->
    </node>

    <node name="offboard_node" pkg="itm_nonlinear_mpc" type="offboard_sim_node.py" respawn="false" clear_params="true" output="screen" launch-prefix="zsh -c 'sleep $(arg node_start_delay1); $0 $@'">
  </node>
</launch>